custom-login/README.md

```diff
@@ -30,15 +30,15 @@ npm install
 Now you need to gather the following information from the Okta Developer Console:
 
 - **Client Id** - The client ID of the SPA application that you created earlier. This can be found on the "General" tab of an application, or the list of applications.  This identifies the application that tokens will be minted for.
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+- **Issuer** - This is the URL of the authorization server that will perform authentication, containing your custom your Okta OrgURL, which can be found in the upper right of your developer console home page. (This is not the same as the URL you access your dev console at.) Once you copy/paste, it should look like: `https://dev-1234.oktapreview.com/oauth2/default`.
 
 Now place these values into the file `src/.samples.config.js` that was created for you in this project:
 
 ```javascript
 export default {
   oidc: {
     clientId: '{clientId}',
-    issuer: 'https://{yourOktaDomain}.com/oauth2/default',
```

Comment:
The value you deleted is the standard syntax we use to specific this.

Reply:
Any ideas to provide clarity for new devs on the difference between Okta Domain, dev console domain, and Org URL? I pasted my dev console domain and got a 404.

Reply:
You can add a note like this:

**TIP:** The value of `{yourOktaDomain}` should be something like `dev-123456.oktapreview`. Make sure you don't include `-admin` in the value!

Reply:
I can do that (although it wouldn't have helped me since I didn't read the comments before filling in the values.) Would it also make sense to change the "Org URL" in the dashboard to "Your Okta Domain" for clarity?